bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/TypeChecker.java

```diff
@@ -588,6 +588,11 @@ public static Object getAnnotValue(TypedescValue typedescValue, String annotTag)
         if (!(describingType instanceof BAnnotatableType)) {
```

Comment:
@warunalakshitha, this method is no longer called, right? The one with BString called instead. Can we remove this? Or can we at least stop updating this for new changes?

Reply:
+1 to remove